src/components/accordion/accordion-test.stories.tsx

```diff
@@ -86,3 +92,549 @@ export const Grouped = () => (
+  );
+};
+
+export const AccordionGroupWithError = () => {
+  const [errors] = React.useState({
+    one: errorVal,
+    two: errorVal,
+    three: errorVal,
+  });
+
+  return (
+    <div
+      style={{
+        marginTop: "16px",
+      }}
+    >
+      <AccordionGroup>
+        <Accordion title="Heading" error={errors.one}>
+          <div style={{ padding: "8px" }}>
+            <Checkbox label="Add error" error={!!errors.one} />
+          </div>
+        </Accordion>
+      </AccordionGroup>
+    </div>
+  );
+};
+
+export const AccordionGroupWithWarning = () => {
+  const [warnings] = React.useState({
+    one: warningVal,
+  });
+
+  return (
+    <div
+      style={{
+        marginTop: "16px",
+      }}
+    >
+      <AccordionGroup>
+        <Accordion title="Heading" warning={warnings.one}>
+          <div style={{ padding: "8px" }}>
+            <Checkbox label="Add warning" warning={!!warnings.one} />
+          </div>
+        </Accordion>
+      </AccordionGroup>
+    </div>
+  );
+};
+
+export const AccordionGroupWithInfo = () => {
+  const [infos] = React.useState({
+    one: infoVal,
+  });
+
+  return (
+    <div
+      style={{
+        marginTop: "16px",
+      }}
+    >
+      <AccordionGroup>
+        <Accordion title="Heading" info={infos.one}>
+          <div style={{ padding: "8px" }}>
+            <Checkbox label="Add info" info={!!infos.one} />
+          </div>
+        </Accordion>
+      </AccordionGroup>
+    </div>
+  );
+};
+
+export const AccordionGroupComponent = () => {
+  return (
+    <AccordionGroup>
+      <Accordion title="First Accordion" onChange={() => {}} width="100%">
+        <Box p={2}>
+          <Textbox label="Textbox in an Accordion" />
+        </Box>
+      </Accordion>
+      <Accordion title="Second Accordion" onChange={() => {}} width="100%">
+        <Box p={2}>
+          <Box height="100px" bg="primary" />
+        </Box>
+      </Accordion>
+      <Accordion title="Third Accordion" onChange={() => {}} width="100%">
+        <div>Content</div>
+      </Accordion>
+    </AccordionGroup>
+  );
+};
+
+export const DynamicContent = () => {
+  const [contentCount, setContentCount] = React.useState(3);
+  const modifyContentCount = (modifier: number) => {
+    if (modifier === 1) {
+      setContentCount(contentCount + 1);
+    }
+    if (modifier === -1 && contentCount > 0) {
+      setContentCount(contentCount - 1);
+    }
+  };
+  return (
+    <>
+      <Button data-element="add-content" onClick={() => modifyContentCount(1)}>
+        Add content
+      </Button>
+      <Button
+        data-element="remove-content"
+        onClick={() => modifyContentCount(-1)}
+        ml={2}
+      >
+        Remove content
+      </Button>
+      <Accordion title="Title" defaultExpanded>
+        {Array.from(Array(contentCount).keys()).map((value) => (
+          <div key={value}>Content</div>
+        ))}
+      </Accordion>
+    </>
+  );
+};
+
+// stories from storybook to import
```

Comment:
Are these used anywhere? They're imported from `./accordion.stories` in the mdx file